src/devices/Bmxx80/Bmxx80Base.cs

```diff
@@ -157,6 +159,7 @@ public void Reset()
         /// Contains <see cref="double.NaN"/> otherwise.
         /// </param>
         /// <returns><code>true</code> if measurement was not skipped, otherwise <code>false</code>.</returns>
+        [Obsolete("TryReadTemperature is deprecated, please use Read or ReadAsync instead.")]
```

Comment:
IMO you should remove Obsolete attribute, there is nothing wrong with this method as is.

Reply:
same for other places

Reply:
Yeah that's fine by me. My only concern was that the new Read method can achieve the same result as these methods and often do it faster so I would discourage using them.